R/series.R

```diff
@@ -75,6 +78,16 @@ dySeries <- function(dygraph,
                      fillGraph = NULL,
                      drawPoints = NULL,
                      pointSize = NULL,
+                     pointShape = c("dot",
```

Comment:
The convention for `dySeries` is to have all of the arguments be `NULL` (implying they accept whatever the global `dyOptions` default is).